evm_loader/program/src/instruction/transaction_step.rs

```diff
@@ -61,41 +60,67 @@ pub fn do_continue<'a>(
     }
 
     let account_storage = ProgramAccountStorage::new(accounts)?;
-    let (mut backend, mut evm) = if reset {
-        storage.reset_steps_executed();
-
-        let mut backend = ExecutorState::new(&account_storage);
-        let evm = Machine::new(storage.trx(), storage.trx_origin(), &mut backend, None)?;
-        (backend, evm)
-    } else {
-        deserialize_evm_state(&storage, &account_storage)?
-    };
+    let (steps_executed, results) = {
+        allocate_or_reinit_state(&account_storage, &mut storage, reset)?;
 
-    let (result, steps_executed, _) = match backend.exit_status() {
-        Some(status) => (status.clone(), 0_u64, None),
-        None => evm.execute(step_count, &mut backend)?,
-    };
+        let mut state_data = storage.read_executor_state();
+        let mut evm = storage.read_evm();
+        //let (mut state, mut evm): (RefMut<'_, ExecutorStateData>, RefMut<'_, Evm>) = (storage.state(), storage.evm());
```

Comment:
Should this line be deleted, or is it left as a comment?

Reply:
done